.github/workflows/pr-title-linter.yml

```diff
@@ -0,0 +1,105 @@
+name: PR Title Checker
```

Comment:
We can now avoid copy-pasting this logic and use a github action instead like [here](https://github.com/warp-ds/component-classes/pull/7#pullrequestreview-1341058736)

Reply:
Yup, done. 🙂